compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEUtils.cpp

```diff
@@ -107,6 +107,218 @@ FailureOr<unsigned> getTilingScaleFactor(Type elemType) {
   return 64 / bitWidth;
 }
 
+// Utility to match iterator type and indexing map for a linalg.generic that
+/// is basically implementing a matmul with 2D input/output operands. Such
+/// matmul variants we get from Pad pipeline.
```

Comment:
Remove comments about "Pad pipeline" and any other pipeline which we have already deprecated.